charts/devlake/templates/statefulsets.yaml

```diff
@@ -162,6 +166,10 @@ spec:
 #          envFrom:
 #            - configMapRef:
 #                name: {{ include "devlake.fullname" . }}-config
+#            {{- if .Values.option.useConnectionDetailsSecret }}
```

Comment:
Please delete instead of commenting out the dead code :)

Reply:
This part was commented when I started working on it. I assumed it was to keep it in case the psql is supported again. But I can remove it if you prefer :)